CODING.md

```diff
@@ -64,6 +64,20 @@ Note: Chrome in headless mode doesn't support the WebGL `EXT_frag_depth`
 extension. So rendering may differ and some bugs can only be present in headless
 mode.
 
+### Useful commands for continuous testing
+
+If you wish to have unit tests to continuously run, you can append to your
+running command `-- --watch`. You will be getting in this case `npm run
+test-unit -- --watch`.
+
+If you want to work on a single test and debug it, without having all the extra
+output, you can use this command:
+```sh
+./node_modules/.bin/cross-env BABEL_DISABLE_CACHE=1 \
+  ./node_modules/.bin/mocha --require @babel/register \
+  test/unit/3dtiles.js -- --watch # replace 3dtiles.js with the correct filename
```

Comment:
Would it then be a good idea to add a `base-test-unit` NPM script that would call mocha without any "target", and rewrite `test-unit` as: `npm run base-test-unit -- test/unit`? then this command could become `npm run base-test-unit -- --watch test/unit/3dtiles.js`.

Reply:
Yeah why not, this seems like a good idea